MerlinAU.sh

```diff
@@ -6412,8 +6422,7 @@ _PostUpdateEmailNotification_()
    #--------------------------------------------------------------
    while [ "$curWaitDelaySecs" -lt "$maxWaitDelaySecs" ]
    do
-      if _WAN_IsConnected_ && \
-         [ "$(nvram get ntp_ready)" -eq 1 ] && \
+      if [ "$(nvram get ntp_ready)" -eq 1 ] && \
```

Comment:
No longer required and causes issues on my node. We could always add an additional value to the options other than "2" but then the accuracy is decreased anyways.

Reply:
This also kinda comes back to the point I was making the other day about this is why I generally shy away from ncram values due to the differences between routers and configurations hahaha.

I shrugged this off thinking it's all good cause it's only 2 ncram values and bamn! Found a configuration that doesn't follow the rule hahaha!! 🤣

Reply:
> This also kinda comes back to the point I was making the other day about this is why I generally shy away from nvram values due to the differences between routers and configurations hahaha.
> 
> I shrugged this off thinking it's all good cause it's only 2 nvram values and bamn! Found a configuration that doesn't follow the rule hahaha!! 🤣
>
>
So far, it looks like it's when the router is functioning as an AiMesh node that some NVRAM vars have different meanings, regardless of the router model. IMO, this is a flaw in the design of AiMesh because I see no technical reason to change the meanings of standard NVRAM values such as those for WAN connection.

Reply:
> > This also kinda comes back to the point I was making the other day about this is why I generally shy away from nvram values due to the differences between routers and configurations hahaha.
> > I shrugged this off thinking it's all good cause it's only 2 nvram values and bamn! Found a configuration that doesn't follow the rule hahaha!! 🤣
> 
> So far, it looks like it's when the router is functioning as an AiMesh node that some NVRAM vars have different meanings, regardless of the router model. IMO, this is a flaw in the design of AiMesh because I see no technical reason to change the meanings of standard NVRAM values such as those for WAN connection.

I tend to agree. I don't work for ASUS but really it would make handling these routers simpler if the meaning of the values didn't change like this I would think?

Reply:
> > > This also kinda comes back to the point I was making the other day about this is why I generally shy away from nvram values due to the differences between routers and configurations hahaha.
> > > I shrugged this off thinking it's all good cause it's only 2 nvram values and bamn! Found a configuration that doesn't follow the rule hahaha!! 🤣
> > 
> > 
> > So far, it looks like it's when the router is functioning as an AiMesh node that some NVRAM vars have different meanings, regardless of the router model. IMO, this is a flaw in the design of AiMesh because I see no technical reason to change the meanings of standard NVRAM values such as those for WAN connection.
> 
> I tend to agree. I don't work for ASUS but really it would make handling these routers simpler if the meaning of the values didn't change like this I would think?
>
From a S/W dev. point of view, it makes coding, testing, debugging & keeping functional requirements much easier when the meaning of variables & their values are always consistent across device models and the different configurations available to the user. Otherwise, one has to keep track of what an NRAM key-pair "means" based on the current "model" or "configuration" being coded/debugged/tested, which makes it more prone to human errors. That may be why the ASUS AiMesh feature has been "buggy" from the start and still is after all these years, based on posts from the forums.

Reply:
@Martinski4GitHub 

DON'T Get me started bud ;)